src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +19,75 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const paginationSerializer: GetCustomersRequestHandler = (req, res, next) => {
+    const PaginationParams = req.query;
+    let offset = 0;
+    let limit = 50;
+  
+    if (PaginationParams.offset) {
+      offset = Number(PaginationParams.offset);
+    }
+  
+    if (PaginationParams.limit) {
+      limit = Number(PaginationParams.limit);
+    }
+  
+    res.locals.paginationParamsSerializer = { offset, limit };
+  
+    next();
+};
+
+const getCustomersSerializer: GetCustomersRequestHandler = (req, res, next) => {
+    const customers = res.locals.getCustomer;
+    res.locals.customerToRespond = [];
+
+    customers.map((customers) => {
+        res.locals.customerToRespond.push({
+            uuid: customers.uuid,
+            name: customers.name,
+            contact: {
+                email: customers.email,
+                phone: customers.phone,
+            },
+            document: {
+                cpf: customers.cpf,
+                cnpj: customers.cnpj,
+            },
+            createdAt: customers.createdAt.toISOString(),
+            updatedAt: customers.updatedAt.toISOString(),
+        });
+    });
```

Comment:
Poderia simplificar e atribuir diretamente, ao invés de criar um array vazio e fazer push.
```suggestion
    res.locals.customerToRespond = customers.map(customer => ({
        uuid: customer.uuid,
        name: customer.name,
        contact: {
            email: customer.email,
            phone: customer.phone,
        },
        document: {
            cpf: customer.cpf,
            cnpj: customer.cnpj,
        },
        createdAt: customer.createdAt.toISOString(),
        updatedAt: customer.updatedAt.toISOString(),
    }));
```